src/components/NetworkSelectDropdown.tsx

```diff
@@ -9,12 +9,13 @@ import { DEFAULT_NETWORK } from '@config';
 import { Typography, Selector, Tooltip } from '@components';
 
 interface Props {
-  network: NetworkId;
+  network?: NetworkId;
   accountType?: WalletId;
   className?: string;
   showTooltip?: boolean;
   disabled?: boolean;
   onChange(network: NetworkId): void;
+  filter?(network: Network): boolean;
```

Comment:
Any reason to filter inside the component rather the modify the prop?

Reply:
`networks` isn't a prop. It comes from `NetworkContext`.